autoscale_cloudroast/test_repo/autoscale/system/integration/test_system_integration_rcv3.py

```diff
@@ -92,6 +95,15 @@ def setUpClass(cls):  # nopep8 - ignore N802
 
         lb_pools = [{'loadBalancerId': cls.pool.id, 'type': 'RackConnectV3'}]
 
+        init_rc_node_ids = []
```

Comment:
Non-blocking nitpick:  :)  there are cool list comprehensions below that do the same thing, which could also be done here.